docker-compose.yaml

```diff
@@ -2,12 +2,12 @@ version: '2'
 services:
     db:
```

Comment:
I propose we rename the "db" container to "influxdb" now that we have multiple databases in play.